### ISSUE TITLE
[11.0] Room view was broken by PR #111

### ISSUE THREAD

**Issue by mclaeysb** [OCA/vertical-hotel#116]
After updating our code and updating the `hotel` module, our 'Room' and 'Room Type' views were seemingly broken by #111 which was written to fix #51 .

For context: 

An error while updating the Hotel app.
```sh
2020-07-27 15:35:57,094 7217 WARNING lignon11-test odoo.schema: Table 'hotel_room_type': unable to set NOT NULL on column 'product_categ_id'!
```

The error while accessing the `Room`view

```sh

File "/home/odoo11/oca/vertical-hotel/hotel/models/hotel_room_type.py", line 47, in name_get
    return [(cat.id, " / ".join(reversed(get_names(cat)))) for cat in self]
File "/home/odoo11/oca/vertical-hotel/hotel/models/hotel_room_type.py", line 47, in <listcomp>
    return [(cat.id, " / ".join(reversed(get_names(cat)))) for cat in self]
TypeError: sequence item 0: expected str instance, bool found
```
The state of our database
![Screen Shot 2020-07-27 at 17 54 57](https://user-images.githubusercontent.com/2920134/88563548-659dbd00-d032-11ea-8b1c-b00a303ab106.png)
![Screen Shot 2020-07-27 at 17 55 08](https://user-images.githubusercontent.com/2920134/88563563-69c9da80-d032-11ea-8072-07291b21c580.png)

**Comment by mclaeysb** [comment]
I think it also broke the room selector wizard in reservations!
@JayVora-SerpentCS

**Comment by JayVora-SerpentCS** [comment]
@dhruvitaSerpentCS can you check?

**Comment by mclaeysb** [comment]
Any news on this? Thanks for checking

I also think this makes a good point for maintaining the OCA practice of 2 positive reviews, including an external one before merging. Let's keep the code quality high together ;)

**Comment by mclaeysb** [comment]
Hi. @JayVora-SerpentCS and @dhruvitaSerpentCS . Would it be possible to take a look at this? This is hindering our further development of the module. Thanks!

**Comment by JayVora-SerpentCS** [comment]
@dhruvitaSerpentCS this is urgent as its late, please do the needful.
Thanks.

**Comment by mclaeysb** [comment]
Seem to be fixed with #120. Thanks for the response!
